runtime/src/snapshot_package.rs

```diff
@@ -228,18 +180,10 @@ impl SnapshotArchiveInfoGetter for SnapshotPackage {
     }
 }
 
+/// Snapshots come in two flavors, Full and Incremental.  The IncrementalSnapshot has a Slot field,
+/// which is the incremental snapshot base slot.
```

Comment:
nit: base -> base full snapshot

Reply:
Is your suggestion to rename "incremental snapshot base slot" or "incremental snapshot base full snapshot slot"?

If yes, then I do not think we want to include "full snapshot". There are two contexts where incremental snapshots are talked about. One is when an incremental snapshot is on its own (i.e. when taking an incremental snapshot, or when getting snapshot storages for an incremental snapshot), and Two is when an incremental snapshot is talked about in reference to a full snapshot (i.e. when loading a bank from both a full and incremental snapshot, or `bank_to_incremental_snapshot_archive()`).

But first, we have `IncrementalSnapshotArchiveInfo` and `FullSnapshotArchiveInfo`, which are passed around and set by various functions. The `IncrementalSnapshotArchiveInfo` has an additional field called `base_slot`, so it's like the incremental snapshot's base slot. When I named this, I felt this communicated the idea of "incremental"; in my head this asks "what slot is this incremental snapshot based on?". The answer is the last full snapshot slot, yes. The comment on the field hopefully is clear:

https://github.com/solana-labs/solana/blob/6a53ec28e03f0e6368e3719daa50166bc21e979d/runtime/src/snapshot_archive_info.rs#L87-L97

So when constructing or using an `IncrementalSnapshotArchiveInfo`, you'll see params named `incremental_snapshot_base_slot`. (like building the incremental snapshot archive path, or getting snapshot storages for an incremental snapshot.)

And when using incremental snapshots with the context of a full snapshot also, you'll see params called `full_snapshot_slot`. Like when calling `bank_to_incremental_snapshot_archive`, or `bank_from_snapshot_archives`.

Hopefully this makes sense! Let me know what you think.

Reply:
makes sense, thanks for the explanation!